### DISCUSSION TITLE
`tensorflow-rocm` Release

### DISCUSSION THREAD

**Discussion by acxz** [rocm-arch/rocm-arch#972] [Announcements]
tensorflow-rocm hasn't been given a lot of love compared to its pytorch counterpart, but it still is around!

tensorflow-rocm-2.13-0-4 has been updated and does build!

Feel free to try it out and report any issues!

**Comment by acxz** [comment]
@tpkessler Would it be possible to integrate tensorflow-rocm into `[community]`?

**Reply by tpkessler** [reply]
It's on my list for the ROCm 5.7.0 release. On the Arch side we're currently having the problem that tensorflow doesn't build with gcc or clang. So we first need to fix this before I can add its ROCm version.

**Reply by acxz** [reply]
Glad to hear! Feel free to take a look at the tensorflow-rocm pkgbuild, i think we have some gcc issues patched, don't know if those are the same ones you guys are running into.
